nomad-cluster.tf

```diff
@@ -1,6 +1,6 @@
 variable "enable_nomad" {
   description = "Nomad builder fleet is used for CCIE v2. It is disabled by default."
```

Comment:
Shouldn't the description now say "enabled by default"?

Reply:
true, thanks, fixed